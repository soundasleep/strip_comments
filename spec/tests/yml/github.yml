name: Ruby

on:
  push:
    branches:
      - main

  pull_request:

# comment
jobs:
  build:
    runs-on: ubuntu-latest # comment

    strategy:
      matrix:
        ruby:
          - 3.0.2   # comment