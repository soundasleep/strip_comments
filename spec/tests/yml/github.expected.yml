name: Ruby

on:
  push:
    branches:
      - main

  pull_request:


jobs:
  build:
    runs-on: ubuntu-latest 

    strategy:
      matrix:
        ruby:
          - 3.0.2   